sdk/servicebus/service-bus/tests.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         RESOURCE_GROUP: $(service-bus-test-resource-group)
         SERVICEBUS_CONNECTION_STRING: $(service-bus-test-connection-string)
       # Service Bus tests do not support concurrent execution
-      MaxParallel: 1
+      MaxParallel: 0
```

Comment:
Remove this line to keep these files DRY. A default value for the parameter is already set in the template file (https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/jobs/archetype-sdk-tests.yml#L4)